examples/jakstat_signaling/mainJakstatSignaling.m

```diff
@@ -50,12 +50,12 @@
 % al.
 
 [exdir,~,~]=fileparts(which('mainJakstatSignaling.m'));
-try
-    amiwrap('jakstat_pesto','jakstat_pesto_syms', exdir);
-catch ME
-    warning('This PESTO example uses the AMICI toolbox (available at https://github.com/ICB-DCM/AMICI). Unfortunately, there was a problem with AMICI when trying to run this example file. Please check if AMICI is properly installed to run this example. The original error message was:');
-    rethrow(ME);
-end
+% try
```

Comment:
reenable?

Reply:
Thanks, that's the kind of stuff that happens while testing... -.-*